src/ComputerPlayer.java

```diff
@@ -1,9 +1,23 @@
+/**
+ * CSC3465(Software Design) - Debugging Lab 
+ * --> "Practicing Debugging with Pig!"
+ * 
+ * @revised by Xu Yan (Brandon)
+ *
+ */
 
 public class ComputerPlayer extends Player{
 	private final int MIN_POINTS = 15;
 	
 	public ComputerPlayer(){
 		super("R2D2");
+		/* 
+		 * [*** Bug Fixes 5. ***]
+		 * 
+		 * this error is caused by the subclass ComputerPlayer doesn't assign the myName variable,
+		 * which is inherited from the super class Player
+		 * */
+		myName = "R2D2";
```

Comment:
Ok, but this change should really happen in the Player class, not here.  That's why the name is passed to the super constructor.